include/APIClient/Router.hpp

```diff
@@ -5,24 +5,27 @@
 
 using PutFormData = std::map<std::string, std::string>;
 
+namespace APIClient {
 class Router {
 public:
+    enum class Route : uint8_t {
+        getAllVulkanInfo,
+        getVulkanInfoWithId,
+        putVulkanInfo
+    };
+
+    static constexpr const char* baseAPIUri = LUGBENCH_API_URI;
+    static constexpr const char* apiVersion = LUGBENCH_API_VERSION;
+    static constexpr const char* apiVersionUri = "api/";
```

Comment:
The class enum/typedef/static variables and class methods should have a different block like this:
`public:
    enum class Route : uint8_t {
        // ....
    };
    // ....

public:
    Route() = default;
    // ....`